packages/fxa-auth-server/test/local/routes/sign.js

```diff
@@ -35,7 +35,7 @@ describe('/certificate/sign', () => {
     );
     return SessionToken.create({
       uid: uuid.v4({}, Buffer.alloc(16)).toString('hex'),
-      email: 'foo@example.com',
+      email: 'vbudhram+mozilla@mozilla.com',
```

Comment:
Q: did you want your LDAP in here? (seems like we have a few LDAP emails in test cases, but you'd be the only active one).
```sh
git grep -IPhio "\b[a-z0-9+\.]+@mozilla\.com\b" -- ":(exclude)packages/fxa-shared/metrics/glean/*.yaml" | sort -i | uniq -c
```